org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug286935TestLanguageGrammarAccess.java

```diff
@@ -156,10 +159,10 @@
 		private final Keyword cTEXTUALTextualKeyword_3_0 = (Keyword)cTEXTUALEnumLiteralDeclaration_3.eContents().get(0);
 		
 		//enum StateType:
-		//	NORMAL | PSEUDO='cond' | REFERENCE='reference' | TEXTUAL='textual';
+		//	NORMAL | PSEUDO = 'cond' | REFERENCE = 'reference' | TEXTUAL = 'textual';
```

Comment:
@ArneDeutsch Missing formatting rule for `EnumLiteralDeclaration` - the space around the `=` is unexpected.